__tests__/unit/src/xcoobee/sdk/Consents.spec.js

```diff
@@ -1,227 +1,227 @@
-const jest = require('jest');

```

Comment:
Please revert the line-endings back to what they were. It's too hard to see the real changes in this file and several others. It also makes reviewing the repo's history difficult.